lib/tasks/solr.rake

```diff
@@ -1,13 +1,26 @@
 # frozen_string_literal: true
+require 'etda_explore/solr_admin'
+require_relative '../../app/models/fake_solr_document'
 
 namespace :solr do
     desc 'Load Fixtures' 
     task load_fixtures: :environment do
-      # TODO refactor
 
-      docs = JSON.parse(File.open('spec/fixtures/current_fixtures.json').read)
-      Blacklight.default_index.connection.add(docs)
-      Blacklight.default_index.connection.commit
+      20.times do 
+        doc = FakeSolrDocument.new
+        Blacklight.default_index.connection.add(doc.doc)
+        Blacklight.default_index.connection.commit
```

Comment:
@whereismyjetpack wonder if we can commit once at the end? It does not matter with this many docs though

Reply:
That was my thought, wouldn't be too hard to move the commit out of the loop. Also we might want to do something like `bundle exec rake solr:load_fixtures[100]` to add more and then the commit will matter. thanks!

Reply:
oh I liked this `bundle exec rake solr:load_fixtures[100]` idea very much!